modules/developer_manual/examples/core/webdav_api/trashbin/list-files-in-trashbin-success-response.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns">
+  <d:response>
+    <d:href>/remote.php/dav/trash-bin/admin/</d:href>
+    <d:propstat>
+      <d:prop>
+        <d:resourcetype>
+          <d:collection/>
+        </d:resourcetype>
+      </d:prop>
+      <d:status>HTTP/1.1 200 OK</d:status>
+    </d:propstat>
+    <d:propstat>
+      <d:prop>
+        <oc:trashbin-original-filename/>
+        <oc:trashbin-original-location/>
+        <oc:trashbin-delete-timestamp/>
+        <d:getcontentlength/>
+      </d:prop>
+      <d:status>HTTP/1.1 404 Not Found</d:status>
+    </d:propstat>
+  </d:response>
+  <d:response>
+    <d:href>/remote.php/dav/trash-bin/admin/255/</d:href>
+    <d:propstat>
+      <d:prop>
+        <oc:trashbin-original-filename>Level 1</oc:trashbin-original-filename>
+        <oc:trashbin-original-location>Level 1</oc:trashbin-original-location>
+        <oc:trashbin-delete-timestamp>1562660798</oc:trashbin-delete-timestamp>
```

Comment:
please remove timestamp from the examples - use datetime instead.

Reply:
More than happy to update the examples when the response from the endpoint changes.

Reply:
the respsone changes based on what you request.

simply request trashbin-delete-datetime instead of trashbin-delete-timestamp and you will get it

Reply:
Wonderful. Thanks for the clarification.